controller_manager/package.xml

```diff
@@ -13,7 +13,12 @@
   <depend>controller_interface</depend>
   <depend>controller_manager_msgs</depend>
   <depend>hardware_interface</depend>
+  <depend>launch</depend>
```

Comment:
Since you are adding many dependencies here. Does it make sense to move `launch_utils.py` into a new package?

Reply:
Maybe, although they are quite basic dependencies. 

I put it here because the original spawner was in this repo. But it could be somewhere else.

@bmagyar thoughts?

Reply:
not speaking for Bence here, but I think it makes sense to have the spawner and its dependencies live in the controller manager repo. Given that the launch components are rather independent from the rest of the code, we can factor this out into its own package later on quite easily.